src/locales/en.json5

```diff
@@ -115,6 +115,7 @@
         "update_publish": "Publish Draft",
         "update_draft_delete": "Abandon Draft",
         "attachment_update": "Update Attachments",
+        "submission_export": "Download Submissions",
```

Comment:
The release criteria specified "Submissions Export", but I thought a verb would nicely match the other actions. I used "Download" instead of "Export", since that's the word used for that button.